### PR TITLE
remove oauth server dependency from most integration tests

### PR DESCRIPTION
**Pull request by deads2k**
/assign liggitt enj

@bparees I think this breaks the back of the problem.  There are stragglers in the origin integration tests that I'll try to mop up at some later date, but this gives you back a working token.

@smarterclayton fyi

### CODE REVIEW

**Comment by liggitt** [comment]
/lgtm
All the utilities were unused?

**Comment by deads2k** [comment]
> /lgtm
> All the utilities were unused?

yeah, weird, right?

File: test/util/client.go [hunk 1]
```diff
@@ -20,10 +23,10 @@ import (
 
 	configapi "github.com/openshift/origin/pkg/cmd/server/api"
 	cmdutil "github.com/openshift/origin/pkg/cmd/util"
-	"github.com/openshift/origin/pkg/cmd/util/tokencmd"
 	oauthapi "github.com/openshift/origin/pkg/oauth/apis/oauth"
 	oauthclient "github.com/openshift/origin/pkg/oauth/generated/internalclientset"
 	"github.com/openshift/origin/pkg/serviceaccounts"
+	userapi "github.com/openshift/origin/pkg/user/apis/user"
 	userclient "github.com/openshift/origin/pkg/user/generated/internalclientset"
```
Comment (bparees) [inline test/util/client.go]: still dependent on internal apis and auth, i guess i'm not seeing how i copy this pattern.

Comment (deads2k) [reply test/util/client.go]: > still dependent on internal apis and auth, i guess i'm not seeing how i copy this pattern.

Hrm. All our integration tests are still on internal.  I don't think I've done anything special that's internal only.  Can you copy it and change the package imports to the externals?

Comment (bparees) [reply test/util/client.go]: maybe.  it's bitten me in other places but i can try it.

**Comment by enj** [comment]
That's one way to force a valid token 😄

**Comment by bparees** [comment]
@deads2k fyi this appears to have let me move forward in image-registry, thanks!

**Pull request merged**